lib/stdlib/src/beam_lib.erl

```diff
@@ -55,7 +55,7 @@
 
 -type beam() :: module() | file:filename() | binary().
 
--type forms()     :: [erl_parse:abstract_form()].
+-type forms()     :: [erl_parse:abstract_form() | erl_parse:form_info()].
```

Comment:
Shouldn't this commit be applied independently of this PR and as soon as possible (e.g. also in 'maint') ?